dashboard/app/models/user.rb

```diff
@@ -216,7 +216,7 @@ def save_email_preference
         email: email,
         opt_in: email_preference_opt_in.downcase == "yes",
         ip_address: email_preference_request_ip,
-        source: EmailPreference::ACCOUNT_SIGN_UP,
+        source: :ACCOUNT_SIGN_UP,
```

Comment:
I believe there was a robust discussion about this, so I don't want to derail, but it does seem a bit strange to me to not see such generic names scoped in some way, by way of having a class prefix.

Reply:
I tend to agree, actually, now that I see this in use. Certainly for IDE autocompletion, but also for readability, I agree the scoped constant is more clear.